piksi_tools/console/settings_view.py

```diff
@@ -38,54 +33,19 @@
 from .settings_list import SettingsList
 from .utils import resource_filename
 
-SETTINGS_REVERT_TIMEOUT = 5
-SETTINGS_RETRY_TIMEOUT = 10
-BATCH_WINDOW = 10
+# TODO: maybe something more elegant..
+import sys
+sys.path.insert(0, 'lib/libsettings')
+# sys.path.insert(0, '/mnt/users/pasi/swiftnav/libsettings/python')
+
+from libsettings import Settings, settings_write_res_e   # noqa: E402
```

Comment:
Not sure I understand the `settings_write_res_e` naming, using lower case with underbars is for functions,  right?  Would suggest `SettingsWriteResponseCodes` or just `ResponseCodes`.

Reply:
Renamed to `SettingsWriteResponseCodes`

Reply:
👍